### PR TITLE
[zh-tw] new translation: Web/JavaScript/Reference/Statements/for...of

### PR DESCRIPTION
**Pull request by BlueBoy247**
### Description

I created a new page to translate [this page](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/for...of) into Traditional Chinese.

### Motivation

I was looking up information about this statement before and found that this documentation hasn't been translated into Traditional Chinese, so I translated it myself, hoping to help more readers.

### Additional details

No.

### Related issues and pull requests

No.

### CODE REVIEW

File: files/zh-tw/web/javascript/reference/statements/for...of/index.md [hunk 1]
```diff
@@ -0,0 +1,374 @@
...
+
+## 語法
+
+```js-nolint
+for (variable of iterable)
+  statement
+```
+
+- `variable`
+  - : 在每次迭代中從序列得到一個值，可以是用 [`const`](/zh-TW/docs/Web/JavaScript/Reference/Statements/const)、[`let`](/zh-TW/docs/Web/JavaScript/Reference/Statements/let) 或 [`var`](/zh-TW/docs/Web/JavaScript/Reference/Statements/var) 宣告，也可以是[賦值](/zh-TW/docs/Web/JavaScript/Reference/Operators/Assignment)目標（例如先前宣告的變數、物件屬性或[解構賦值模式](/zh-TW/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment)）。使用 `var` 宣告的變數不是迴圈的局部變數，即它們與 `for...of` 迴圈位於同一作用域中。
```
Comment (jasonren0403) [inline files/zh-tw/web/javascript/reference/statements/for...of/index.md:22]:
```suggestion
  - : 在每次迭代中從序列得到一個值，可以是用 [`const`](/zh-TW/docs/Web/JavaScript/Reference/Statements/const)、[`let`](/zh-TW/docs/Web/JavaScript/Reference/Statements/let) 或 [`var`](/zh-TW/docs/Web/JavaScript/Reference/Statements/var) 宣告的變數，也可以是[賦值](/zh-TW/docs/Web/JavaScript/Reference/Operators/Assignment)目標（例如先前宣告的變數、物件屬性或[解構賦值模式](/zh-TW/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment)）。使用 `var` 宣告的變數不是迴圈的局部變數，即它們與 `for...of` 迴圈位於同一作用域中。
```

**Review by jasonren0403 — COMMENTED** [review]
👍 还不错，只有一个小地方需要修改

File: files/zh-tw/web/javascript/reference/statements/for...of/index.md (hunk 1 above)
Comment (BlueBoy247) [reply files/zh-tw/web/javascript/reference/statements/for...of/index.md]: 感謝指正🙇‍♂️，已修改

**Review by jasonren0403 — APPROVED** [review]

**Review by jasonren0403 — APPROVED** [review]
感谢贡献，欢迎来到MDN 👏

**Pull request merged**